libs/application/templates/parental-leave/src/fields/Rights/RequestDaysSlider.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useState } from 'react'
+import { FieldBaseProps, getValueViaPath } from '@island.is/application/core'
+import { Controller, useFormContext } from 'react-hook-form'
+import { Box, Text } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import Slider from '../components/Slider'
+import BoxChart, { BoxChartKey } from '../components/BoxChart'
+import config from '../../config'
+
+const RequestDaysSlider: FC<FieldBaseProps> = ({ field, application }) => {
+  const maxDays = config.maxDaysToGiveOrReceive
+  const { id } = field
+  const { formatMessage } = useLocale()
+  const currentAnswer = getValueViaPath(
+    application.answers,
+    field.id,
+    1,
```

Comment:
I might be totally wrong, but is this `1` related to the `minPeriodDays` config value? should `currentAnswer` be minimum 14 days? I don't see the `minPeriodDays`  used anywhere

Reply:
Good point, I'm not sure if there is a min days that you can give or request. I will confirm this one with Parental Leave.
I added the const in case we needed it later for the periods validation.